docs/src/pages/customization/overrides.md

```diff
@@ -1,7 +1,6 @@
 # Overrides
 
-As components evolve in different contexts, Material-UI supports four different types of customization needs going from
-**the most specific** to **the most generic**:
+As components evolve in different contexts, Material-UI supports four different types of customization needs going from **the most specific** to **the most generic**:
```

Comment:
I meant to ask about this. Are the components "evolving" in different contexts in this example, or simply being *used* in different contexts?

Reply:
used

Reply:
Done.